Package.swift

```diff
@@ -20,14 +20,7 @@ let package = Package(
         .target(
             name: "Valet",
             dependencies: []),
-        .target(
-            name: "LegacyValet",
-            dependencies: [],
-            publicHeadersPath: "Public"),
-        .testTarget(
-            name: "ValetTests",
-            dependencies: ["Valet"])
     ],
     swiftLanguageVersions: [.v4, .v4_2, .v5]
 )
-let version = Version(3, 2, 5)
```

Comment:
Whoops. We need to remember to keep this version up-to-date.

Note that I'm not bumping the podspec version in this PR, since this PR does not affect the Cocoapods installation.